seedpod_ground_risk/layers/osm_tag_layer.py

```diff
@@ -17,29 +21,18 @@ def query_osm_polygons(osm_tag, bound_poly: sg.Polygon) -> gpd.GeoDataFrame:
     :param shapely.Polygon bound_poly: bounding box around requested area in EPSG:4326 coordinates
     """
     from time import time
-    import requests
 
     t0 = time()
     bounds = bound_poly.bounds
-    overpass_url = "https://overpass.kumi.systems/api/interpreter"
-    query = """
-              [out:json]
-              [timeout:30]
-              [bbox:{s_bound}, {w_bound}, {n_bound}, {e_bound}];
-              (
-                  node[{tag}];
-                  way[{tag}];
-                  rel[{tag}];
-              ); 
-              out center body;
-              >;
-              out center qt;
-          """.format(tag=osm_tag,
-                     s_bound=bounds[0], w_bound=bounds[1], n_bound=bounds[2], e_bound=bounds[3])
-    resp = requests.get(overpass_url, params={'data': query})
-    if resp.status_code != 200:
-        print(resp)
-    data = resp.json()
+    overpass_urls = ["https://overpass.kumi.systems/api/interpreter", "https://lz4.overpass-api.de/api/interpreter",
```

Comment:
Why not add all the [global instances](https://wiki.openstreetmap.org/wiki/Overpass_API#Public_Overpass_API_instances)? Also `kumi` endpoint is in here twice.